TreeTracker/SwiftGen/Assets.swift

```diff
@@ -54,6 +54,7 @@ internal enum Asset {
     }
   }
   internal enum Colors {
+    internal static let backgroundGreen = ColorAsset(name: "BackgroundGreen")
```

Comment:
I think when we spoke we decided to remove the green background to keep the UI consistent with the rest of the app.

Reply:
Removed all traces of green background color